### PR TITLE
update issue template

### PR DESCRIPTION
**Pull request by dustinxie**
# Description
as title

## Type of change
Please delete options that are not relevant.
- [x] Code refactor or improvement

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -4,25 +4,16 @@ about: Create a bug report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 <!-- Please answer these questions when reporting a bug. Thanks! -->
 
-### What version of iotex-core are you using (`ioctl version`)?
```
Comment (dustinxie) [inline .github/ISSUE_TEMPLATE/bug_report.md:12]: `ioctl version` not accurate, it gives the version of ioctl binary installed on local machine, but not the current running image/code

**Review by raullenchai — APPROVED** [review]

**Review by Liuhaai — APPROVED** [review]

**Review by Newman1515 — COMMENTED** [review]
Approve

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Newman1515) [reply .github/ISSUE_TEMPLATE/bug_report.md:12]: Ok

**Review by Newman1515 — COMMENTED** [review]
Ok

**Review by Newman1515 — COMMENTED** [review]
Fr

**Review by Newman1515 — COMMENTED** [review]
Ok

**Pull request merged**